en/developer_guide/custom_content_attribute_display.rst

```diff
@@ -0,0 +1,74 @@
+Customize Content Attribute Display
+===================================
+
+Context
+-------
+
+To give the minimum constraint on contents, Open Orchestra allows you to store any type
+of value in the contentAttribute.
+Sometimes, there is no easy way to render the value to the backoffice user so we have added
+a stringValue.
+The stringValue is a projection of the value in a string.
```

Comment:
An example like displaying a mongo entity would be much more explicit about what is the usage of a StringValue.
Why aren't the first letters of the class names not capitalized ?